packages/recommend-core/src/utils/__tests__/mapToRecommendations.test.ts

```diff
@@ -0,0 +1,129 @@
+import { mapToRecommendations } from '../mapToRecommendations';
+
+const response = {
+  results: [
+    {
+      exhaustiveNbHits: true,
+      hits: [
+        {
+          objectID: '1-1',
+          name: 'Product 1-1',
+          _score: 100,
+        },
+        {
+          objectID: '1-2',
+          name: 'Product 1-2',
+          _score: 90,
+        },
+        {
+          objectID: '1-3',
+          name: 'Product 1-3',
+          _score: 70,
+        },
+        {
+          objectID: '1-4',
+          name: 'Product 1-4',
+        },
+        {
+          objectID: '1-5',
+          name: 'Product 1-5',
+        },
+      ],
+      hitsPerPage: 10,
+      nbHits: 10,
+      nbPages: 1,
+      page: 0,
+      processingTimeMS: 1,
+    },
+    {
+      exhaustiveNbHits: true,
+      hits: [
+        {
+          objectID: '2-1',
+          name: 'Product 2-1',
+          _score: 100,
+        },
+        {
+          objectID: '2-2',
+          name: 'Product 2-2',
+          _score: 90,
+        },
+        {
+          objectID: '2-3',
+          name: 'Product 2-3',
+          _score: 70,
+        },
+        {
+          objectID: '2-4',
+          name: 'Product 2-4',
+        },
+        {
+          objectID: '2-5',
+          name: 'Product 2-5',
+        },
+      ],
+      hitsPerPage: 10,
+      nbHits: 10,
+      nbPages: 1,
+      page: 0,
+      processingTimeMS: 1,
+    },
+  ],
+};
+
+describe('mapToRecommendations', () => {
```

Comment:
We can add a test to ensure that it doesn't sort a well-sorted response

Reply:
As discussed in DM, the test already covers this case.